Library/Formula/packer-windows-plugins.rb

```diff
@@ -0,0 +1,152 @@
+    url "https://code.google.com/p/go.crypto",
+      :revision => "69e2a90ed92d",
+      :using => :hg
+  end
+  go_resource "github.com/hashicorp/go-version" do
+    url "https://github.com/hashicorp/go-version.git",
+      :revision => "bb92dddfa9792e738a631f04ada52858a139bcf7"
+  end
+  go_resource "github.com/mitchellh/iochan" do
+    url "https://github.com/mitchellh/iochan.git",
+      :revision => "b584a329b193e206025682ae6c10cdbe03b0cd77"
+  end
+  go_resource "github.com/mitchellh/mapstructure" do
+    url "https://github.com/mitchellh/mapstructure.git",
+      :revision => "442e588f213303bec7936deba67901f8fc8f18b1"
+  end
+  go_resource "github.com/dylanmei/iso8601" do
+    url "https://github.com/dylanmei/iso8601.git",
+      :revision => "2075bf119b58e5576c6ed9f867b8f3d17f2e54d4"
+  end
+  go_resource "github.com/masterzen/winrm" do
+    url "https://github.com/masterzen/winrm.git",
+      :revision => "813d86ee814a2d07cb1153d0c1cb922f3f8239b7"
+  end
+  go_resource "github.com/masterzen/simplexml" do
+    url "https://github.com/masterzen/simplexml.git",
+      :revision => "95ba30457eb1121fa27753627c774c7cd4e90083"
+  end
+  go_resource "github.com/masterzen/xmlpath" do
+    url "https://github.com/masterzen/xmlpath.git",
+      :revision => "13f4951698adc0fa9c1dda3e275d489a24201161"
+  end
+  go_resource "github.com/nu7hatch/gouuid" do
+    url "https://github.com/nu7hatch/gouuid.git",
+      :revision => "179d4d0c4d8d407a32af483c2354df1d2c91e6c3"
+  end
+  go_resource "github.com/mitchellh/go-fs" do
+    url "https://github.com/mitchellh/go-fs.git",
+      :revision => "faaa223588dd7005e49bf66fa2d19e35c8c4d761"
+  end
+  go_resource "github.com/going/toolkit" do
+    url "https://github.com/going/toolkit.git",
+      :revision => "6185c1893604d52d36a97dd6bb1247ace93a9b80"
+  end
+  go_resource "github.com/mitchellh/go-vnc" do
+    url "https://github.com/mitchellh/go-vnc.git",
+      :revision => "f54c4812027006edcb7fbbb78f5d013105fe2c53"
+  end
+  go_resource "github.com/hashicorp/go-msgpack" do
+    url "https://github.com/hashicorp/go-msgpack.git",
+      :revision => "71c2886f5a673a35f909803f38ece5810165097b"
+  end
+  go_resource "github.com/hashicorp/yamux" do
+    url "https://github.com/hashicorp/yamux.git",
+      :revision => "b2e55852ddaf823a85c67f798080eb7d08acd71d"
+  end
+  go_resource "github.com/rakyll/command" do
+    url "https://github.com/rakyll/command.git",
+      :revision => "0f2fed130caff9721655936b8a71d98b5252fc3a"
+  end
+  go_resource "github.com/racker/perigee" do
+    url "https://github.com/racker/perigee.git",
+      :revision => "44a7879d89b7040bcdb51164a83292ef5bf9deec"
+  end
+  go_resource "github.com/mitchellh/gophercloud-fork-40444fb" do
+    url "https://github.com/mitchellh/gophercloud-fork-40444fb.git",
+      :revision => "40444fbc2b10960682b34e6822eb9179216e1ae1"
+  end
+
+
+  def install
+    pluginpath = Pathname.new("#{ENV['HOME']}/.packer.d/plugins")
```

Comment:
Homebrew never intentionally does this. We don't let formulae create directories in ` $HOME ` and it will actually become impossible to do so in the near future.